### ISSUE TITLE
GCONPROD not running using GRAT

### ISSUE THREAD

**Issue by mspringer64** [OPM/opm-simulators#1689]
I added a simple network to the SPE3 model and tried to add the GCONPROD keyword. However, using GRAT as control mode the Flow simulator does not run. Using ORAT the results from Eclipse and Flow are identical. The two .DATA files are attached as .txt files.

[SPE3CASE1_GRAT.txt](https://github.com/OPM/opm-simulators/files/2712527/SPE3CASE1_GRAT.txt)
[SPE3CASE1_ORAT.txt](https://github.com/OPM/opm-simulators/files/2712528/SPE3CASE1_ORAT.txt)
![opm_error](https://user-images.githubusercontent.com/5927277/50481976-b2e42380-09e4-11e9-8d0c-2b6b81ed9dde.png)
![fopr](https://user-images.githubusercontent.com/5927277/50482057-19694180-09e5-11e9-9865-dbe14ebde76b.png)

**Comment by atgeirr** [comment]
This looks like a simple bug, I will try a small modification and see if that is sufficient.

**Comment by GitPaean** [comment]
Probably, you can try to add GRAT to 
```
 485     void WellsGroup::applyProdGroupControls()
 486     {
 487         ProductionSpecification::ControlMode prod_mode = prodSpec().control_mode_;
 488         switch (prod_mode) {
 489         case ProductionSpecification::ORAT:
 490         case ProductionSpecification::WRAT:
 491         case ProductionSpecification::LRAT:
 492         case ProductionSpecification::RESV:
```

**Comment by atgeirr** [comment]
I tested that, and it seems to work. I do however get some time step cutting, and oscillation in the well. It is hard to see if the Ecl output above suffers from the same, the oscillation is quite small, see the uploaded picture:

[oscillatingwell.pdf](https://github.com/OPM/opm-simulators/files/2720962/oscillatingwell.pdf)

I will make a small PR to change the buggy line.

**Comment by mspringer64** [comment]
I made a figure which shows the WOPR on the same scale than the oscillatingwell.pdf. Eclipse has no oscillation behavior. I also included the time step in the figure. For the ECL the time stepping is one month. OPM seems to chop time steps. I am not concerned with the difference in ORAT. However, the chopping in time steps due to oscillations might be a problem in real size simulations
![prod_grat](https://user-images.githubusercontent.com/5927277/50631897-62217d00-0f46-11e9-9251-7be185f9e8c7.png)
